wee_slack.py

```diff
@@ -2536,6 +2536,10 @@ def subprocess_thread_message(message_json, eventrouter, channel, team):
             if parent_message.thread_channel:
                 parent_message.thread_channel.buffer_prnt(message.sender, text, message.ts, tag_nick=message.sender_plain)
 
+            if config.thread_messages_in_channel:
+                channel.buffer_prnt(
+                    message.sender, '[{}] {}'.format(parent_message.hash, text), message.ts, tag_nick=message.sender_plain)
```

Comment:
I think we should colorize the prefix, in the same color as the suffix, i.e. `w.color(config.thread_suffix_color)`.